Mathlib/Analysis/Meromorphic/NormalFormAt.lean

```diff
@@ -198,7 +316,7 @@ theorem meromorphicNFAt_toMeromorphicNFAt :
       split_ifs with h₃f
       · obtain ⟨h₁G, _, h₃G⟩ := Classical.choose_spec ((hf.order_eq_int_iff 0).1 (h₃f ▸ hn.symm))
         apply Filter.EventuallyEq.eq_of_nhds
-        apply h₁G.eventuallyEq_nhdNE_iff_eventuallyEq_nhd (by fun_prop)
+        apply (h₁G.eventuallyEq_nhdNE_iff_eventuallyEq_nhd (by fun_prop)).1
```

Comment:
`apply (lemma : _ \iff _).1` is better spelled `rw [lemma]`. Same below

Reply:
Done elsewhere but not here. With the `rw` tactic, the clause `by fun_prop` no longer works.